contrib/test/integration/main.yml

```diff
@@ -54,6 +54,8 @@
   tags:
     - e2e
   tasks:
+    - name: install Golang tools
```

Comment:
Why do we need this here?

Reply:
Cause otherwise we'll run with ami golang which today was still 1.8.4. This should actually remove the caching from e2e also